packages/suite/src/hooks/wallet/useSendFormImport.ts

```diff
@@ -41,12 +51,22 @@ export const useSendFormImport = ({ network, tokens, localCurrencyOption, fiatRa
 
                 if (currency === network.symbol) {
                     // csv amount in crypto currency
-                    output.amount = item.amount || '';
+                    const cryptoValue = item.amount || '';
+                    if (!isInteger(cryptoValue) && shouldSendInSats) {
+                        // try to convert to satoshis
+                        output.amount = amountToSatoshi(cryptoValue, network.decimals);
+                    } else {
+                        output.amount = cryptoValue;
+                    }
+
                     // calculate Fiat from Amount
                     if (fiatRates && fiatRates.current) {
+                        const fiatAmount = shouldSendInSats
+                            ? formatAmount(output.amount, network.decimals)
+                            : output.amount;
                         output.fiat =
                             toFiatCurrency(
-                                output.amount,
+                                fiatAmount,
```

Comment:
I think the naming is off here (the value represents crypto amount) and perhaps it can be simplified just by using `cryptoValue` as the first argument of `toFiatCurrency`).

Reply:
well, `cryptoValue` (crypto input) could be represented  either in bitcoin or satoshi units while `toFiatCurrency` always requires bitcoin units, this is why it needs to be conditionally transformed